Fixed #3023 for Graphs tab sub menu

ddb4github opened the pull request.

Commit:
Fixed #3023 for Graphs tab sub menu

netniV reviewed (changes requested):
this needs a changelog entry

ddb4github commented:
will change changelog as soon with non-good grammar

Commit:
Fixed#3023
	Update ChangeLog
	Add global event to auto-close Graph/Ellipsis/Help/User menu
	Potential JS error for previous about var 'hideTabId'

Commit:
Fixed#3023: Update ChangeLog

netniV commented on include/layout.js:
```diff
@@ -2564,15 +2567,23 @@ function setupEllipsis() {
+		if($(event.target).parents('.menuoptions').length == 0 && $('.menuoptions').is(':visible')) {
+			$('.menuoptions').slideUp(120);
+		}
+	});
```
My understanding is that this will now trigger whenever a user clicks, as long as they are not in that menu.  Is that the intended purpose?

ddb4github replied:
Yes, line#2580 and line#2583 will be always called once user click.
